frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/forcepushdialog/VcsPushDialogAdapterHolder.java

```diff
@@ -0,0 +1,71 @@
+package com.virtuslab.gitmachete.frontend.actions.forcepushdialog;
+
+import javax.swing.JRootPane;
+
+import com.intellij.dvcs.push.PushSupport;
+import com.intellij.dvcs.push.VcsPushOptionValue;
+import com.intellij.dvcs.push.ui.VcsPushDialog;
+import com.intellij.dvcs.repo.Repository;
+import com.intellij.openapi.project.Project;
+import org.checkerframework.checker.guieffect.qual.UIEffect;
+import org.checkerframework.checker.initialization.qual.UnderInitialization;
+import org.checkerframework.checker.initialization.qual.UnknownInitialization;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+/**
+ * The holden adapter is required to pass custom {@link GitForcePushDialog} to {@link com.intellij.dvcs.push.PushController}.
```

Comment:
Texas hold 'em? :trollface:

Reply:
See comment to GitForcePushDialog.java, possibly you can get rid of this class completely